cookbooks/zarr-stac-report/data-consumers/index.qmd

```diff
@@ -0,0 +1,98 @@
+
+Both of those access patterns should be supported by tooling, but depending on how the catalog is set up some patterns may be simpler and faster than others (at this point in time). So which is better? Basically it depends on the structure of the Zarr store(s) and the STAC catalog that you are interacting with.
+
+## One big Zarr store in a standalone STAC collection
+
+The STAC catalog contains a collection for each Zarr store and there are collection-level assets that point to the location of the Zarr store. There are no items at all in this setup.
+
+In this scenario any STAC metadata exists purely for discovery and cannot be used for filtering or subsetting (see [Future Work](#future-work) for more on that). To search the STAC catalog to find collections of interest you will use the [Collection Search API Extension][7]. Depending on the level of metadata that has been provided in the STAC catalog you can search by the name of the collection and possibly by the variables (aka groups) contained within the Zarr store.
+
+### Read straight to xarray
+
+Once you have found the collection of interest, the best approach for accessing the data is to construct the lazily-loaded data cube in xarray (or an [xarray.DataTree][8] if the Zarr store has more than one group) and filter from there.
+
+To do this you can use the `zarr` backend directly or you can use [the `stac` backend][1] to streamline even more. The `stac` backend is mostly useful if the STAC collection uses the xarray extension.
+
+Constructing the lazy data cube is likely to be very fast if there is a [consolidated metadata file][9] OR the data is in Zarr-3 and the Zarr metadata fetch is highly parallelized ([read more][4]).
+
+
+## Many small Zarr stores in STAC items
+
+The STAC catalog contains many items in each collection and the items each contain one asset pointing to a Zarr store. In this setup, the Zarr store represents a particular area in time and space (a scene). This is very similar to the common practice of using COGs and STAC where each COG represents one band at one scene. The difference is instead of one band per COG, all bands are in one Zarr store (this is conceptually equivalent to a multi-band COG).
+
+In this scenario the STAC metadata can be used to find data that overlaps the region of interest (both in time and space) before constructing a lazily-loaded data cube in xarray.
+
+### Filter in STAC first
+
+If you are interacting with a STAC API and the Zarr store is cataloged in STAC following [`xstac`][10] conventions then all the metadata you need for filtering and subsetting is available at the STAC level so you should be able to search either in the web-browser directly or using pystac-client.
```

Comment:
how would I know it's xstac conventions as a user?

Reply:
Mentioning Data Cube Extension again.